extensions/mapstruct/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -25,9 +25,8 @@ name: "Camel MapStruct"
 description: "Type Conversion using Mapstruct"
 icon-url: "https://camel.apache.org/_/img/logo-d-f21b25ba38.svg"
 metadata:
-  unlisted: true
   guide: "https://camel.apache.org/camel-quarkus/latest/reference/extensions/mapstruct.html"
   categories:
   - "integration"
   status:
```

Comment:
mapstruct pr has been mixed with xlst saxon one ?

Reply:
Hmm, it was add by `mvn process-resources -Pformat -N`. So I think it shoud be excluded in this PR. Thanks for pointing it out!

Reply:
Remove.